detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/SolutionInspector.cs

```diff
@@ -130,9 +130,9 @@ public Container GetContainer()
                                 while (fileNotFound)
                                 {
                                     parentPath =
-                                        parentPath.Substring(0, parentPath.LastIndexOf("/"));
+                                        parentPath.Substring(0, OperatingSystem.IsWindows() ? parentPath.LastIndexOf("\\") : parentPath.LastIndexOf("/"));
                                     string checkFile = Path.Combine(parentPath, projectRelativePath);
-                                    if (parentPath.Equals("/"))
+                                    if (parentPath.Equals("/") || parentPath.Equals("\\"))
```

Comment:
I'm still a bit unsure about this approach. Do paths here always start with either "/" or "\\"?
Could it be something like "C:\\" in Windows? If so, it would be safer to make the comparison to the path root: https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getpathroot?view=net-7.0#system-io-path-getpathroot(system-string)